src/index.js

```diff
@@ -1,10 +1,80 @@
 // @flow
 
+/*  eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Cookies from 'js-cookie';
 
 import App from './App';
 import enLocale from '../i18n/en/translation.json';
+import { Requester } from './helpers/Requests';
+import type { User } from './types';
+
+type Props = {||};
+
+type State = {|
+  user: User,
+  loginToken: string | null,
+|};
+
+class Root extends React.Component<Props, State> {
+  cookieKey: string;
+
+  constructor(props) {
+    super(props);
+
+    this.cookieKey = 'mockedLogin';
+
+    this.state = {
+      user: null,
```

Comment:
In order to be able to keep the session after refresh when testing on github pages what if we:
1. Move the `user` in Line 37 out of the class.
2. Then replace this block in the constructor with:
```
    const loginToken = Cookies.get(this.cookieKey);
    this.state = {
      user: loginToken ? user : null,
      loginToken,
    };
```